source/material_model/depth_dependent.cc

```diff
@@ -57,57 +57,58 @@ namespace aspect
 
 
 
-    template <int dim>
-    void
-    DepthDependent<dim>::read_viscosity_file(const std::string &filename,
-                                             const MPI_Comm &comm)
-    {
-      /* This method is used for the Table method of depth dependent viscosity */
-      // Read data from disk and distribute among processes
-      std::istringstream in(Utilities::read_and_distribute_file_content(filename, comm));
-
-      double min_depth=std::numeric_limits<double>::max();
-      double max_depth=-std::numeric_limits<double>::max();
-
-      /* The input viscosity file has two columns that are the viscosity and Depth */
-      std::string header;
-      std::getline(in, header);/* Discard header line */
-      std::vector<double> visc_vec;
-      std::array< std::vector<double>, 1 > depth_table;
-      while (!in.eof())
-        {
-          double visc, depth;
-          in >> depth;
-          in >> visc;
-          if ( in.eof() ) break;
-          min_depth = std::min(depth, min_depth);
-          max_depth = std::max(depth, max_depth);
-          depth_table[0].push_back(depth);
-          visc_vec.push_back( visc );
-        }
-      /* Check to make sure that limits of depth-dependence table encompass entire domain. */
-      AssertThrow( min_depth <= 0.0, ExcMessage("Minimum depth in viscosity file must be <= 0.0") );
-      AssertThrow( max_depth >= this->get_geometry_model().maximal_depth(),
-                   ExcMessage("Maximum depth in viscosity file must be >= maximal depth of model") );
-
-      Table<1,double> viscosity_table( depth_table[0].size() );
-
-      for ( unsigned int i=0; i<visc_vec.size(); i++)
-        {
-          viscosity_table[i] = visc_vec[i];
-        }
-      viscosity_file_function
-        = std_cxx14::make_unique<Functions::InterpolatedTensorProductGridData<1>>(depth_table, viscosity_table);
-    }
+    // template <int dim>
```

Comment:
Please remove all commented code. We do not need it, because if you want to go back to an earlier version you can always do `git revert COMMIT` to make a new commit that exactly undoes what you did in commit id COMMIT, or `git reset COMMIT` (optional: `git reset --hard COMMIT`) if you want to change your branch (with --hard: and checked out files) to exactly the state they were at commit id COMMIT, losing all later commits unless you have a different branch that still has them.